tests/cases/fourslash/codeFixSpellingVsImport.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference path='fourslash.ts' />
+
+// Tests that the import fix is returned first.
+
+// @Filename: /a.ts
+////export const foo = 0;
+
+// @Filename: /b.ts
+////const foof = 0;
+////foo;
+
+goTo.file("/b.ts");
+verify.codeFixAvailable([
+    { description: `Import 'foo' from module "./a"` },
+    { description: "Change spelling to 'foof'" },

```

Comment:
Shouldn't we prioritize spelling fix over import here too ?

Reply:
A user actually requested that we not do that: https://github.com/Microsoft/TypeScript/issues/23258#issuecomment-379764815

I think this makes sense -- it's more likely that you misspelled some existing property than that you forgot to declare a different property with nearly the same name. But if you have `import { foo1 } from "mod";`, we don't want to correct all `foo2` to `foo1` if `foo2` actually does exist. The difference in the latter case is we actually do know that `foo2` is a valid name, it just wasn't imported; whereas in the property declaration case we have no evidence that there was ever supposed to be a second property.

Reply:
I generally agree with @krryan's intuition, and I'm okay with experimenting a little bit here. Let's let the import come first.